specs/specTemplate/README.md

```diff
@@ -9,31 +9,32 @@ Brief component description
 
 ## Elements
 
-Screenshot & brief elements description
+All elements of the component and their interactivity will be described, including a visual representation of it (image or animations).
 
 > e.g.
-> Screenshot of the component (http://joxi.ru/bmonQewtMy479r)
+>
+> ![elements](./assets/elements.png)
 > **Toggle** consists of:  **switch**, **icon**, and **background**. The **switch** moves across the **background** in order to set the value to On/Off, and **icon** indicates the state of the value.
 
 
 
 ## API
 
-### Component Props
+#### Component Props
 
 | name     | type    | defaultValue | isRequired | description                              |
 | -------- | ------- | ------------ | ---------- | ---------------------------------------- |
-| disabled | boolean | false        |            | If `true`, the componentName will not be interactive. |
-| label    | string  |              |            | Text to display in accessibility mode.   |
-| name     | string  |              |            | The name of the slider. Behaves like the name attribute of an input element. |
+| disabled | boolean | false        |            | If `true`, the component will not be interactive |
+| label    | string  |              |            | Text to display in accessibility mode    |
+| name     | string  |              |            | The name of the slider. Behaves like the name attribute of an input element |
 | error    | bool    | FALSE        |            | Sets the `:error` CSS state on the `<slider>` |
 | rtl      | bool    | FALSE        |            | Makes the component RTL                  |
 
 
 
-#### Accepted Children ( if applicable )
+#### Accepted Children / Predefined Roles  ( if applicable )
```

Comment:
Arnon, my main concern with the naming is the potential confusion between the children here (which are the "data-slot", and the Subcomponents/Custom Pseudo Selectors, which are also called "Children" later in the spec.

Whatever naming convention we choose, we should avoid this.

Alternately, we could go with the solution of passing these children (from this bullet) as props, then we'd have one section less and one less confusion to worry about. Take a look at Vova's PR for that.